web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,39 +1,61 @@
 import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+// TODO: remember that all exisiting metadata will have to be updated
+
+/** Signed metadata AKA "Additional Info" */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // regen:C01-Project
+
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
   'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
   'regen:offsetGenerationMethod': string;
   // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
   'http://regen.network/offsetGenerationMethod'?: string;
   'regen:projectSize': ProjectSize;
   'regen:projectStartDate': TypeValue;
   'regen:projectEndDate': TypeValue;
-  'regen:boundaries': TypeValue;
-  'regen:creditClass': CreditClass;
-  'regen:landManagementActions': LandManagementActions;
+}
+
+/** This is unsigned metadata */
+export interface ProjectPageMetadataLD {
+  '@context': Context;
+  '@type': string; // regen:C01-Project-Page
+
+  // media
+  'schema:image': UrlType;
+  'schema:creditText': string;
   'regen:galleryPhotos': UrlList;
   'regen:previewPhoto': UrlType;
   'regen:videoURL': UrlType;
-  'regen:glanceText': any;
+
+  // roles
+  'regen:projectDeveloper': ProjectStakeholder;
+  'regen:landSteward': ProjectStakeholder;
+  'regen:landOwner': ProjectStakeholder;
+  'regen:projectOriginator': ProjectStakeholder;
+
+  // marketing
   'regen:landStory': string;
   'regen:landStewardStory': string;
   'regen:landStewardPhoto': UrlType;
   'regen:projectQuote': ProjectQuote;
   'regen:landStewardStoryTitle': string;
+  'schema:description'?: string;
+
+  // legacy? - may not exist in standards repo
+  'regen:boundaries': TypeValue;
+  'regen:creditClass': CreditClass;
+  'regen:landManagementActions': LandManagementActions;
+  'regen:glanceText': any;
 }
```

Comment:
@blushi @S4mmyb @clevinson 

After talking with @wgwz, we think  `ProjectPageMetadataLD` is useful naming for what we were calling "non-queryable" metadata. This maintains some context across repos. Queryable metadata would remain `ProjectMetadataLD`. WDYT?

Also, the fields you see in this interface represent our best understanding of where we landed in our last meeting together. Do you agree these should all be items that can be edited without signing a ledger TX? 

Let's discuss here if you have any evolving thoughts on any of these. Otherwise, @wgwz and I will continue in this direction. Thanks!

Reply:
I believe that looks good.
What about `regen:projectOperator` for CFC projects? on project pages, this data appears under "project developer" label

Reply:
I wondered about that too. I think it should remain in the "signed" data. Even though it is similar to the roles, it seems very coupled to the CFC project.

Reply:
So after discussing this with @S4mmyb, we should just use project developer in place of project operator in this case.
Also, regarding roles in general, it's preferable to just store them through the existing references in the project table (`developer_id`, `originator_id`, etc.) and create `party`s for them if they don't exist yet (just like it's already done for carbonplus grasslands and kasigau/mai-ndombe projects I believe). Planning ahead for the user profiles stories where project stakeholders could edit their profile and that would be reflected on all the projects they are related to.

Reply:
Ok, so we would remove `regen:projectOperator` from the[ C02 project shape](https://github.com/regen-network/regen-registry-standards/blob/7e32d372161c12b3145fb674e1e0a6613b670119/jsonld/projects/C02-project.jsonld#L43), and reassign the data for those existing projects to the `regen:projectDeveloper`, correct?

Reply:
>Also, regarding roles in general, it's preferable to just store them through the existing references in the project table (developer_id, originator_id, etc.) and create partys for them if they don't exist yet (just like it's already done for carbonplus grasslands and kasigau/mai-ndombe projects I believe). Planning ahead for the user profiles stories where project stakeholders could edit their profile and that would be reflected on all the projects they are related to.

I might be misunderstanding and so it's probably worth talking through it some more. But I don't see how we would be able to do this now. 

Here's the info available to us now for `projectOperator` in the project metadata (to be renamed `projectDeveloper`):
![Screen Shot 2022-12-15 at 3 14 35 PM](https://user-images.githubusercontent.com/10120306/207957963-671049ee-16dd-4f9d-9e6a-9dad27a39991.jpg)

Here's what we require in the party table (i.e. a wallet address):
![Screen Shot 2022-12-15 at 3 16 25 PM](https://user-images.githubusercontent.com/10120306/207958262-015352bb-aebb-4734-bb82-62990664d53c.jpg)

Wouldn't we need to gather up a wallet address to assign for this purpose, if we are going to make use of these references now?

It might be better to wait until we have the account creation process (the login system) in place before we start populating parties

Reply:
I understand your point but wallet_id is not required here.
If the current data structure is gonna stay the same with the login system, mainly parties being referenced in the project table then I believe that's fine to start populating parties.

Reply:
i’ll make some time to review the changes i have in the login PR, i think i just need some time to see how the pieces fit together. i am just thinking about how we were trying to do the whole thing where an account has a corresponding party that has a corresponding wallet (separate tables). and if we put this data in now, for example, how would we know which party needs to be used during the account creation process? i think the entries we put in now, may just end stale data? that would be difficult to manually tie in to the login system?

Reply:
Thinking about the project create flow. 

Our UI adds a party to the postgres DB when we add a Role (project developer). So, if we add more CFC projects in the future, we will have to make sure the Project Developer is added in that Roles step. If we consider this a required field for CFC, we will want to add some validation to enforce it. 

But maybe that is not necessary.

Reply:
Here are some thoughts I've had thinking through the current database model in comparison to the database model in the login system PR. Some of these notes are tangential to our discussion here and we can consider these separately. Please correct me if I make any incorrect statements!

In the current model:
- Each `party` can have many `project` references.
- But each `project` can only have one developer, one originator, etc.
- A single `party` can represent one or more of these roles:
    - a developer, originator, broker, land owner, registry, reseller and steward. do we still need all of these roles?
- The `project` table has a ref to `wallet.id` table, do we still need this ref?
- The `project` table has a ref to `user.id` table called `creator_id`, do we still need this ref?

In the login model:
- A given `party` represents a profile that is tied to a unique `wallet` address.
- A `party` represents a particular `wallet` address and is tied to an `account`.

A mutual goal in both of these models is that a `party` represents a "profile". So indeed, in this way being able to link for example, user profiles, to a particular project can be achieved. But thinking towards when we do implement the login system, we may have to re-think the UI for attaching a particular wallet as a project developer (for example).

TL;DR I think the migration to login model will be fairly tricky. And we should try to do as much clean up as we can in the database to make the model easier to think about.

Reply:
>Q: In the edit project workflow, do we have a way to edit who the project developer is?

Yes. But the code may need some updating.

Reply:
> - The `project` table has a ref to `wallet.id` table, do we still need this ref?
> - The `project` table has a ref to `user.id` table called `creator_id`, do we still need this ref?

I think we might want to keep the former moving forward.